client/components/masonry-wave/style.scss

```diff
@@ -0,0 +1,23 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.masonry-wave__container {
+  display: grid;
+	grid-template-columns: repeat(var(--masonry-wave--amountOfColumns), 1fr);
+	grid-auto-rows: minmax(min-content, max-content);
+	column-gap: 0.75rem;
+	.masonry-wave__column {
+		&:nth-child(even) {
```

Comment:
@Rolilink Looks like we're getting a console warning, in case it's related to this:
`The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".`

Reply:
@allilevine This warning in here is actually coming from the Breadcrumb component in[ here](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/breadcrumb/index.tsx#L19). I will log an issue to fix this one since I will need some time to test that the component is still working fine after adding the change.